integration-tests/scripts/install.sh

```diff
@@ -201,34 +201,62 @@ tpa_integration() {
   fi
 }
 
+artifactory_integration() {
+  if [[ "${registry_config}" == "artifactory" ]]; then
+    echo "[INFO] Configure Artifactory integration into RHTAP"
+    # disable Quay installation
+    yq e '.rhtapCLI.features.redHatQuay.enabled = false' -i "${config_file}"
+
+    ARTIFACTORY_URL="${ARTIFACTORY_URL:-$(cat /usr/local/rhtap-cli-install/artifactory-url)}"
+    ARTIFACTORY_TOKEN="${ARTIFACTORY_TOKEN:-$(cat /usr/local/rhtap-cli-install/artifactory-token)}"
+    ARTIFACTORY_DOCKERCONFIGJSON="${ARTIFACTORY_DOCKERCONFIGJSON:-$(cat /usr/local/rhtap-cli-install/artifactory-dockerconfig-json)}"
+    ./bin/rhtap-cli integration --kube-config "$KUBECONFIG" artifactory --url="${ARTIFACTORY_URL}" --token="${ARTIFACTORY_TOKEN} " --dockerconfigjson="${ARTIFACTORY_DOCKERCONFIGJSON}"
+  fi
+}
+
+nexus_integration() {
+  if [[ "${registry_config}" == "nexus" ]]; then
+    echo "[INFO] Configure Nexus integration into RHTAP"
+    # disable Quay installation
+    yq e '.rhtapCLI.features.redHatQuay.enabled = false' -i "${config_file}"
+
+    NEXUS_URL="${NEXUS_URL:-$(cat /usr/local/rhtap-cli-install/nexus-url)}"
+    NEXUS_DOCKERCONFIGJSON="${NEXUS_DOCKERCONFIGJSON:-$(cat /usr/local/rhtap-cli-install/nexus-dockerconfig-json)}"
+    ./bin/rhtap-cli integration --kube-config "$KUBECONFIG" nexus --url="${NEXUS_URL}" --dockerconfigjson="${NEXUS_DOCKERCONFIGJSON}"
+  fi
+}
+
 install_rhtap() {
   echo "[INFO] Start installing RHTAP"
-  github_integration
   echo "[INFO] Building binary"
   make build
 
   echo "[INFO] Installing RHTAP"
   jenkins_integration
-  gitlab_integration
-  quayio_integration
+  tpa_integration
   acs_integration
+  github_integration
+  gitlab_integration
   bitbucket_integration
-  tpa_integration
+  quayio_integration
```

Comment:
I don't see this function in this file, neither gitlab_integration, lots of them are missing from my eyes. Am I missing something. Damn 🙄

Reply:
Is the assumption is we will only run for the rhtap installed one. No hosted quay?